docstore/driver/codec.go

```diff
@@ -715,8 +715,8 @@ func wrap(err error, code gcerr.ErrorCode) error {
 var fieldCache = fields.NewCache(parseTag, nil, nil)
 
 // Copied from encoding/json, go 1.12.
-func isEmptyValue(v reflect.Value) bool {
-	switch v.Kind() {
+func IsEmptyValue(v reflect.Value) bool {
```

Comment:
Unfortunately encoding/json's definition of "empty value" is not identical to "zero value," and I think we want "zero value". The difference is that maps and slices must be nil, not just have length zero.

Reply:
Never mind, changed my mind.